src/Adapter/AbsAdapter.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbWriter\Synapse\Adapter;
+
+use Keboola\DbWriter\Exception\UserException;
+use Keboola\DbWriter\Synapse\SynapseWriter;
+use MicrosoftAzure\Storage\Blob\BlobRestProxy;
+use MicrosoftAzure\Storage\Common\Exceptions\ServiceException;
+use MicrosoftAzure\Storage\Common\Internal\Resources;
+
+class AbsAdapter implements IAdapter
+{
+    private bool $isSliced;
+
+    private string $region;
+
+    private string $container;
+
+    private string $name;
+
+    private string $connectionEndpoint;
+
+    private string $connectionAccessSignature;
+
+    private string $expiration;
+
+    public function __construct(array $absInfo)
+    {
+        preg_match(
```

Comment:
Import `ABS -> Synapse` je skopirovany/inspirovany z `keboola/php-db-import-export`.
Idealne by bolo pouzivat priamo tuto kniznicu, ale pouziva `Doctrine` a museli by sme tak v celom repe pouzivat `Doctrine` namiesto PDO`. Vzhladom na kratky cas, som to radsej prekopiroval, kde bolo treba.